solutions/src/main/java/ru/sberbank/school/task02/EmptyQuoteException.java

```diff
@@ -0,0 +1,13 @@
+package ru.sberbank.school.task02;
+
+import ru.sberbank.school.task02.exception.FxConversionException;
+
+/**
```

Comment:
First sentence of Javadoc is missing an ending period.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a2079562523>*